manifests/init.pp

```diff
@@ -263,12 +271,12 @@
   validate_string($log_level)
   validate_integer($dogstatsd_port)
   validate_string($statsd_histogram_percentiles)
-  validate_string($statsd_forward_port)
+  validate_re($_statsd_forward_port, '^\d*$')
   validate_string($proxy_host)
-  validate_string($proxy_port)
+  validate_re($_proxy_port, '^\d*$')
   validate_string($proxy_user)
   validate_string($proxy_password)
-  validate_string($graphite_listen_port)
+  validate_re($_graphite__listen_port, '^\d*$')
```

Comment:
Typo here (two underscores). This should be:
`validate_re($_graphite_listen_port, '^\d*$')`